devicetypes/smartthings/zwave-motion-light-sensor.src/zwave-motion-light-sensor.groovy

```diff
@@ -90,10 +89,6 @@ def updated() {
 def configure() {
 	// Device wakes up every deviceCheckInterval hours, this interval allows us to miss one wakeup notification before marking offline
 	sendEvent(name: "checkInterval", value: 2 * deviceWakeUpInterval * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
-
-	if (isEverspringFloodlight()) {
-		response(zwave.configurationV1.configurationSet(parameterNumber: 3, size: 2, scaledConfigurationValue: 600)) //enables illuminance report every 10 minutes
```

Comment:
Generally, if you are removing a device from a DTH that has been published where there are customer devices already using it you would leave code for that device in the DTH and remove the fingerprint. This would be so that devices already paired continue to work. This seems a little less risky, but I'd probably leave it, and leave a note above the `if` block indicating why it is staying around.

Reply:
@tpmanley Thoughts?

Reply:
Since it's in the configure method I don't anticipate it being an issue, since anything already added would have already had this called.